src/Azure/Orleans.Persistence.AzureStorage/Providers/Storage/AzureBlobStorageOptions.cs

```diff
@@ -0,0 +1,107 @@
+﻿using Microsoft.Extensions.Options;
+using Microsoft.WindowsAzure.Storage;
+using Newtonsoft.Json;
+using Orleans.Persistence.AzureStorage;
+using Orleans.Runtime;
+using Orleans.Runtime.Configuration;
+using System;
+using System.Collections.Generic;
+
+namespace Orleans.Hosting
```

Comment:
also this namespace should be Orleans.Configuration :)

Reply:
I will move all options in another PR, this is the same issue as table ect